Readme.md

```diff
@@ -193,6 +193,21 @@ gulp.task('watch', () => {
 ```
 
 
+## Using pnpm for package restore
+
+[pnpm](https://pnpm.io/) is a faster and more efficient package manager. If it's installed globally, then the Gulp Extension module will use that instead of *npm* to restore packages.
+
+### Installation and usage
+
+To install *pnpm* globally run this command: `npm install pnpm -g`. Once it's complete, the module will automatically use that to restore packages.
+
+### Notes
+
+* *pnpm* supports restoring packages directly to a folder so moving the *node_modules* folder to a parent folder is not necessary anymore.
+* *pnpm* uses its own package lock file, meaning you need to maintain both *pnpm-lock.yaml* and *package-lock.json* files in case of adding a new package or updating one.
```

Comment:
```suggestion
* *pnpm* uses its own package lock file, meaning if you want to keep npm compatibility then you need to maintain both *pnpm-lock.yaml* and *package-lock.json* files in case of adding a new package or updating one.
```